src/client/src/apps/SpotlightRoute/Spotlight.tsx

```diff
@@ -51,10 +62,17 @@ export const Spotlight: FC = () => {
       case 'Enter':
         const value = e.currentTarget.value
         dispatch({ type: 'SEND_REQUEST', request: value })
-        setTimeout(
-          () => dispatch({ type: 'RECEIVE_RESPONSE', response: `I heard "${value}".` }),
-          500,
-        )
+        serviceStub.createRequestResponseOperation('nlp', 'getNlpIntent', value)
+          .pipe(
+            timeout(5000),
+            take(1)
+          )
+          .subscribe((response: any) => {
```

Comment:
should we leave a comment to fix `any` in the future (to introduce actual type)?

Reply:
let me add the types for dialogflow